src/main/resources/ath-container/Dockerfile

```diff
@@ -33,9 +33,9 @@ ENV PATH="$PATH:/opt/maven/bin"
 ENV SHARED_DOCKER_SERVICE true
 
 # Allow injecting uid and git to match directory ownership
-ARG uid=1000
+ARG uid=1001
```

Comment:
shouldn't this already get detected properly?
https://github.com/jenkins-infra/pipeline-library/blob/a6261f941d974857a57929c5074955e741e2ba38/vars/runATH.groovy#L100-L102

Reply:
@timja our comments crossed: see my response to Jesse: https://github.com/jenkinsci/acceptance-test-harness/pull/672#discussion_r652974486